packages/suite/src/utils/wallet/__tests__/utxoSortingUtils.test.ts

```diff
@@ -0,0 +1,43 @@
+import { testMocks } from '@suite-common/test-utils';
+
+import { sortUtxos } from '../utxoSortingUtils';
+
+const UTXOS = [
+    testMocks.getUtxo({ amount: '1', blockHeight: undefined, txid: 'txid1', vout: 0 }),
+    testMocks.getUtxo({ amount: '1', blockHeight: undefined, txid: 'txid2', vout: 1 }),
+    testMocks.getUtxo({ amount: '2', blockHeight: 1, txid: 'txid2', vout: 0 }),
+    testMocks.getUtxo({ amount: '2', blockHeight: 2, txid: 'txid3', vout: 0 }),
+];
+
+const ACCOUNT_TRANSACTIONS = [
+    testMocks.getWalletTransaction({ txid: 'txid1', blockTime: undefined }),
+    testMocks.getWalletTransaction({ txid: 'txid2', blockTime: 1 }),
+    testMocks.getWalletTransaction({ txid: 'txid3', blockTime: 2 }),
+];
+
+describe('sortUtxos', () => {
+    it('should sort UTXOs by newest first', () => {
```

Comment:
The test description should not start with `should`. I believe we even have it in Notion somewhere (but I am lazy to search it). 

```suggestion
    it('sorts UTXOs by newest first', () => {
```